### PR TITLE
Expanding CLO status

### PR DESCRIPTION
**Pull request by ewolinetz**
Addresses https://jira.coreos.com/browse/LOG-402

### CODE REVIEW

**Comment by ewolinetz** [comment]
I want to make this dependent on https://github.com/openshift/elasticsearch-operator/pull/130 so that we can expose the new ES stats at the CLO level

File: pkg/k8shandler/status.go [hunk 1]
```diff
@@ -156,18 +165,40 @@ func (clusterRequest *ClusterLoggingRequest) getElasticsearchStatus() ([]logging
...
-				ReplicaSets:   getReplicaSetNames(node.Status),
-				StatefulSets:  getStatefulSetNames(node.Status),
-				Pods:          getPodMap(node.Status),
-				ClusterHealth: node.Status.ClusterHealth,
+				ClusterName:            cluster.Name,
+				NodeCount:              cluster.Spec.Nodes[0].NodeCount,
+				ClusterHealth:          cluster.Status.ClusterHealth,
+				Cluster:                cluster.Status.Cluster,
+				ShardAllocationEnabled: cluster.Status.ShardAllocationEnabled,
+				Pods:              getPodMap(cluster.Status),
```
Comment (richm) [inline pkg/k8shandler/status.go]: formatting

Comment (ewolinetz) [reply pkg/k8shandler/status.go]: weird... that is with `gofmt`

Comment (ewolinetz) [reply pkg/k8shandler/status.go]: moved things around and it formatted it better...

**Pull request merged**